src/main/java/com/educare/unitylend/dao/UserRepository.java

```diff
@@ -13,22 +10,34 @@
 @Repository
 public interface UserRepository {
 
-    @Select(SELECT_USERS)
-    List<User> getAllUsers();
-
-    @SelectProvider(type = UserRepository.class, method = "getUserForUserIdQuery")
-    User getUserForUserId(@Param("userId") String userId);
-
-    static final String SELECT_USERS = "select user_id as userID, password, name from user_table";
-
     static String getUserForUserIdQuery(String userId){
         StringBuilder sqlQueryBuilder = new StringBuilder();
 
-        sqlQueryBuilder.append("select user_id as userID, password, name from user_table where 1=1 ");
+        sqlQueryBuilder.append("select userid as userID, password, name,email,dob,income,officename,collegeuniversity,locality from User where 1=1 ");
```

Comment:
Modify the queries to use mapped variableNames and do not use directly table column names. 
i.e - select userid as userId, password as password , name as name ,email as email ,dob,income,officename as officeName ,collegeuniversity as collegeUniversity